src/handlers/comment/handlers/help.ts

```diff
@@ -41,6 +43,6 @@ export const generateHelpMenu = () => {
     }
   });
 
-  helpMenu += "```";
+  if (!ASSIGN_COMMAND_ENABLED) helpMenu += "```\n***_To assign yourself to this issue, you should open a draft pull request that is linked._***";
```

Comment:
```suggestion
  if (!ASSIGN_COMMAND_ENABLED) helpMenu += "```\n***_To assign yourself to an issue, please open a draft pull request that is linked to it._***";
```

Reply:
Linking might be tricky if projects dont have pull request templates set up with instructions. since this feature is not on by default its probably okay though!